src/ralph/backends/data/async_mongo.py

```diff
@@ -100,7 +100,7 @@ async def list(
 
         try:
             collections = await database.list_collections()
-            for collection_info in collections:
+            async for collection_info in collections:
```

Comment:
Doesn't this mean there was (is) a missing test ? Has this been added ?

Reply:
It shouldn't have work in the first place though, I didn't understand how it could work before.
Can't see how to test this without testing the `motor` package

Reply:
Well spotted! That was surprising to me too.
The current tests should already cover this change.
It seems `motor` fixed the bug we were relying upon in their [3.3.0 release](https://motor.readthedocs.io/en/3.3.0/changelog.html) 5 days ago [here](https://github.com/mongodb/motor/compare/3.2.0...3.3.0#diff-7aaa9d1f95fbcd82603b33bee372d43a6e1398cfe67e082d3ac158e565f2d891R547).
Thus `list_collections` now returns a `AsyncIOMotorCommandCursor` instead of a `CommandCursor`.
As this change seems to be incompatible with previous `motor` versions, could we update `setup.cfg` with `motor[srv]>=3.3.0`?

Reply:
@SergioSim you're right, constraining it to `>=3.3.0` !